packages/ra-core/src/auth/usePermissions.ts

```diff
@@ -42,25 +33,24 @@ const emptyParams = {};
  *     };
  */
 const usePermissions = <Permissions = any, Error = any>(
-    params = emptyParams
-): State<Permissions, Error> => {
-    const [state, setState] = useSafeSetState<State<Permissions, Error>>({
-        isLoading: true,
-    });
+    params = emptyParams,
+    useQueryParams: UseQueryOptions<Permissions, Error> = {
```

Comment:
I'd call it `queryParams` instead of `useQueryParams`

Reply:
done, thanks!